streaming/base/storage/util.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2023 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Utilities for storage."""
+
+import os
+from typing import Any, Dict
+
+import requests
+
+
+def get_databricks_unity_catalog_volume_info(volume_name: str) -> Dict[str, Any]:
+    """Returns the Databricks Unity Catalog volume info.
+
+    Args:
+        volume_name (str): The name of the Databricks Unity Catalog volume.
+
+    Returns:
+        Dict[str, Any]: The volume info.
+    """
+    import requests
+
+    db_host = os.environ['DATABRICKS_HOST']
```

Comment:
Can we check if `DATABRICKS_HOST` and  `DATABRICKS_TOKEN` is present in `os.environ` and if it is not, then throw a ValueError with a descriptive error message ?

Reply:
Reverted this